isodata/nyiso.py

```diff
@@ -189,24 +203,24 @@ def get_historical_lmp(
         end=None,
         market: str = None,
         locations: list = None,
+        location_type: str = None,
     ):
         """
-        Supported Markets: REAL_TIME_5_MIN, DAY_AHEAD_5_MIN
+        Supported Markets: REAL_TIME_5_MIN, DAY_AHEAD_HOURLY
+
+        Supported Location Types: ZONE, GENERATOR
```

Comment:
is it supposed to "zone" and "generator" all lowercase?

Reply:
yes the strings are lowercase but they were used so frequently I put them as constants at the top of the file. is it less confusing to write out the strings here?

Reply:
ya, because the strings are what a user needs to pass to the function